target/linux/ath79/dts/ar7161_adtran_bsap1880.dtsi

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar7100.dtsi"
+
+/ {
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-boot = &led_green_status;
+		led-failsafe = &led_yellow_status;
+		led-running = &led_green_status;
+		led-upgrade = &led_yellow_status;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wlan5g {
+			label = "bluesocket:green:wifi5g";
+			gpios = <&gpio 8 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy1tpt";
+		};
+
+		wlan2g {
+			label = "bluesocket:green:wifi2g";
+			gpios = <&gpio 7 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		led_green_status: green_status {
+			label = "bluesocket:green:status";
+			gpios = <&gpio 5 GPIO_ACTIVE_LOW>;
+		};
+
+		led_yellow_status: yellow_status {
+			label = "bluesocket:yellow:status";
+			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
```

Comment:
Please use `gpio-keys`, it's interrupt driven, so preferred.

Reply:
This part was copied from an existing .dts file. I didn't notice it.